ajax/libs/upng-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "upng-js",
+  "description": "Small, fast and advanced PNG / APNG encoder and decoder",
+  "version": "2.0.0",
+  "author": "photopea",
```

Comment:
```js
"author": {
  "name": "photopea",
  "url": "https://github.com/photopea"
}
```
Let's modify this field as the same with upstream from npm.
This is helpful to people get the info.

Thanks.